internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import "fmt"
+
+func convertNamespaceDefinitionToLongBlob(t *tables) string {
+	return fmt.Sprintf(`ALTER TABLE %s
+		MODIFY serialized_config LONGBLOB NOT NULL;`,
```

Comment:
We sure we don't want MEDIUMBLOB here? 

BLOB = 64KB
MEDIUMBLOB=16MB
LONGBLOB=4GB

64k to 4g seems like a big jump

Reply:
It is merely a max size, still constrained by the overall schema size limits. Further, the other types have overhead: https://www.tutorialspoint.com/What-is-the-maximum-length-of-data-we-can-put-in-a-BLOB-column-in-MySQL

Reply:
that doc is definitely wrong, longblobs [use a 4-byte prefix](https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#:~:text=L%20%2B%204%20bytes%2C%20where%20L%20%3C%20232) (the only distinction between the types is the number of bytes used for the length prefix)

but the 1 byte difference is probably not a big deal, lgtm

Reply:
Darn internet... maybe we use MEDIUMBLOB then?